via/static/scripts/video_player/hooks/test/use-side-by-side-layout-test.js

```diff
@@ -62,14 +63,9 @@ describe('useSideBySideLayout', () => {
   }
 
   function sideBySideState() {
-    const attr = container.getAttribute('data-side-by-side');
-    if (attr === 'true') {
-      return true;
-    } else if (attr === 'false') {
-      return false;
-    } else {
-      return undefined;
-    }
+    return (
+      container.querySelector('[data-testid="side-by-side-enabled"]') !== null
+    );
```

Comment:
This helps test the value which is now returned by the hook.

Reply:
I considered removing it, but then I thought it might be useful for debugging. But I'm open to remove it, don't have a strong opinion.